pkg/service/restore/worker_tables.go

```diff
@@ -0,0 +1,314 @@
+// Copyright (C) 2023 ScyllaDB
+
+package restore
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
```

Comment:
I suggest to change the name of that file.
`worker_tables` indicates that logic related to tables of `worker` struct is kept here, what is not true.

There is new struct tablesWorker.

Please name it `tables_worker`